apps/website/package.json

```diff
@@ -54,7 +54,7 @@
 		"ariakit": "^2.0.0-next.41",
 		"cmdk": "^0.1.20",
 		"meilisearch": "^0.30.0",
-		"next": "^13.0.7-canary.1",
+		"next": "canary",
```

Comment:
Should pin a specific canary version, if you aren't waiting for something specific that is.